packages/nimble-blazor/README.md

```diff
@@ -94,6 +98,22 @@ There are currently extra manual steps required to use the Nimble design tokens
 The SCSS compilation happens before the rest of Blazor's compilation, so this approach works fine with Blazor CSS isolation.  
 Note: This approach requires periodically updating the Nimble tokens SCSS files manually (whenever the Nimble Blazor NuGet version is updated).
 
+### Localization
```

Comment:
For parity's sake I guess we should link to these docs from Storybook in a `## Blazor` section.

Reply:
Done